pkgs/development/libraries/science/math/cutensor/default.nix

```diff
@@ -5,12 +5,12 @@
 
 rec {
   cutensor_cudatoolkit_10_1 = callPackage ./generic.nix rec {
```

Comment:
Cutensor 1.3 dropped support for cudatoolkit 10.1 so this doesn't build. Let's:

* keep the cutensor_cudatoolkit_10_1 definition as-is so it still builds
* put the update to 1.3 in cutensor_cudatoolkit_10_2, and have the subsequent versions override from 10_2